build_cran.sh

```diff
@@ -0,0 +1,13 @@
+OUT_DIR=$1
+CRAN_DIR=$2
+
+echo "build CRAN packages"
+
+while true; do
+    read -p "Build paws with cran packages only (y/n)? " yn
```

Comment:
I think at some point it'd be worth it to add a flag so you didn't need to go through the prompt, because it's not fun to do every time. Doesn't need to be changed now though.

Reply:
Good point, I will add a TODO comment so this point isn't lost